src/format/reveal/format-reveal-theme.ts

```diff
@@ -284,3 +285,9 @@ function quartoLayer(): SassLayer {
 function themeLayer(theme: string): SassLayer {
   return sassLayerFile(theme);
 }
+
+function quartoRevealBrandLayer(): SassLayer {
+  return sassLayerFile(
+    resourcePath(join("formats", "revealjs", "brand", "brand.scss")),
+  );
+}
```

Comment:
Why are we adding `brand.scss` if this is to add it empty ? 

Is this to prepare for future addition ? Or to make reveal similar to bootstrap ? 

Just curious to understand why adding `quartoRevealBrandLayer()` ... 

I would have thought that any brand addition would go in it, but it seems defaults were added on `quartoLayer` directly.

Reply:
I thought it was going to be necessary, so I added it.

I then realized it wouldn't be, but the infrastructure is already there for the future, so I left it there. I will add "currently intentionally blank", but I'm not opposed to removing this either.